browser/ui/views/toolbar/wallet_button.cc

```diff
@@ -48,24 +49,31 @@ content::WebContents* GetActiveWebContents() {
 class BraveWalletButtonHighlightPathGenerator
     : public views::HighlightPathGenerator {
  public:
+  explicit BraveWalletButtonHighlightPathGenerator(bool use_extra_left_margin)
+      : use_extra_left_margin_(use_extra_left_margin) {}
+
   // HighlightPathGenerator:
   SkPath GetHighlightPath(const views::View* view) override {
     DCHECK(view);
 
     gfx::Rect rect(view->size());
     rect.Inset(GetToolbarInkDropInsets(view));
-    rect.Inset(gfx::Insets::TLBR(0, 0, 0, -1 * kBraveWalletLeftMarginExtra));
+    rect.Inset(gfx::Insets::TLBR(
+        0, 0, 0,
+        use_extra_left_margin_ ? -1 * kBraveWalletLeftMarginExtra : 0));
```

Comment:
just FYI, there're Rect::Outset() and gfx::Outsets and I think this is where they can be used. I'm okay with the current code so you can leave it as is.